app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1> # of Jobs by Level of Interest </h1>
+
+<table>
+<th> Interest Level </th>
+<th> # of Jobs </th>
+  <% Job.jobs_by_interest.each do |interest, object| %>
```

Comment:
too much logic in your views (as we talked about earlier!) - put this in your controller and send it in via an instance variable 👍